client/settings/disable-upe-modal/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { useContext, useEffect } from 'react';
+
+import { __ } from '@wordpress/i18n';
+import { dispatch } from '@wordpress/data';
+import { Button, Dashicon, ExternalLink } from '@wordpress/components';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import ConfirmationModal from 'components/confirmation-modal';
+import PaymentMethod from 'components/payment-methods-list/payment-method';
+import useIsUpeEnabled from 'settings/wcpay-upe-toggle/hook';
+import WcPayUpeContext from 'settings/wcpay-upe-toggle/context';
+
+const NeedHelpBarSection = () => {
+	return (
+		<div className="disable-modal-help-notice">
+			<Dashicon className="disable-help-icon" icon="info-outline" />
+			<p>
+				{ interpolateComponents( {
+					mixedString: __(
+						'Need help? Visit {{ docsLink /}} or {{supportLink /}}.',
+						'woocommerce-payments'
+					),
+					components: {
+						docsLink: (
+							// eslint-disable-next-line max-len
+							<ExternalLink href="https://docs.woocommerce.com/document/payments/additional-payment-methods/#introduction">
+								{ __(
+									'WooCommerce Payments docs',
+									'woocommerce-payments'
+								) }
+							</ExternalLink>
+						),
+						supportLink: (
+							// eslint-disable-next-line max-len
+							<ExternalLink href="https://woocommerce.com/contact-us/">
+								{ __(
+									'contact support',
+									'woocommerce-payments'
+								) }
+							</ExternalLink>
+						),
+					},
+				} ) }
+			</p>
```

Comment:
Seeing this not to be well vertically aligned
![Screen Shot 2021-07-22 at 3 32 17 PM](https://user-images.githubusercontent.com/273592/126705765-e7321b98-767d-43f8-86d7-a88bef62d468.png)

And it can wrap under the icon in smaller views

![Screen Shot 2021-07-22 at 3 35 47 PM](https://user-images.githubusercontent.com/273592/126706181-91ddca32-7181-45d1-b892-ea6870fd606c.png)


```
    display: flex;
    align-items: center;
    align-content: center;
    justify-content: flex-start;
    flex-direction: row;
```
on the container can fix it.